packages/alert/package.json

```diff
@@ -32,7 +27,7 @@
   "homepage": "https://github.com/CraveFood/farmblocks#readme",
   "peerDependencies": {
     "prop-types": "^15.7.2",
-    "react": "^16.8.2",
+    "react": "^16.8.3",
```

Comment:
The root `package.json` has `"react": "^16.8.4"`
But inside the packages, it's `"react": "^16.8.3"`